service/src/test/java/greencity/service/ubs/UBSClientServiceImplTest.java

```diff
@@ -1159,9 +1155,8 @@ void saveFullOrderFromLiqPayFailPaidOrder() {
         when(tariffsInfoRepository.findTariffsInfoLimitsByCourierIdAndLocationId(anyLong(), anyLong()))
             .thenReturn(Optional.of(ModelUtils.getTariffInfo()));
         when(bagRepository.findById(3)).thenReturn(Optional.of(bag));
-        when(orderRepository.findById(any())).thenReturn(Optional.of(order));
 
-        Assertions.assertThrows(BadRequestException.class,
+        Assertions.assertThrows(NotFoundException.class,
```

Comment:
I do not see any changes in `ubsService.saveFullOrderToDBFromLiqPay`
Why did you change the exception?